bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -233,9 +236,16 @@ public synchronized void login() {
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
                         "Received invalid data (login)");
             } else if (data.isSuccess()) {
+                if (response.getHeaders().get(HttpHeader.SET_COOKIE) != null) {
+                    List<HttpCookie> cookies = HttpCookie.parse(response.getHeaders().get(HttpHeader.SET_COOKIE));
+                    if (!cookies.isEmpty()) {
+                        cookie = cookies.get(0);
+                        logger.debug("Cookie retrieved");
+                    }
+                }
                 logger.debug("SomfyTahoma version: {}", data.getVersion());
```

Comment:
I don't think the extra cookie handling is necessary. Tahoma needs the cookie as well but the jetty HttpClient handles this out of the box (unlike HttpURLConnection which I used before)